src/snowflake/cli/_plugins/nativeapp/lsp_commands.py

```diff
@@ -51,3 +52,10 @@ def open_app() -> MessageResult:
             return MessageResult(
                 'Snowflake Native App not yet deployed! Please run "runApplication" first.'
             )
+
+    @workspace_command(server, "poc_cmd")
+    def poc_cmd(input: CmdPocInput) -> CmdPocOutput:
+        return CmdPocOutput(
```

Comment:
Note that the default type checking behavior in pycharm and visual studio DOES NOT seem to complain about type violations (the whole point of this).

I got it to behave as expected in pycharm after installing the pydantic plugin. To make sure bugs are detected, we would want to stricter type validation (if not there yet during "build" time) using mypy or equivalent.

![Screenshot 2024-09-25 at 7 33 28 PM](https://github.com/user-attachments/assets/9c104247-d6ec-42e8-b523-f9e3365d24bd)